### DISCUSSION TITLE
How server side angular way pagination API designed?

### DISCUSSION THREAD

**Discussion by RajasekharKp** [l-lin/angular-datatables#1558] [Q&A]
Hi Team, Trying to implement server pagination using Angular Datatables. I took `https://l-lin.github.io/angular-datatables/#/basic/server-side-angular-way` for reference. I can able to load the data in the table but unable to perform `sort, search and pagination` validations. If any chance where can we get the sample code of `API`  is designed for search, sort and paginations and importance of`draw` key in the response. It would be great help if we can get sample code of the example API. Could you please if any reference links,

Thanks in advance.

**Comment by shanmukhateja** [comment]
Hi @RajasekharKp 

This is clearly a duplicate of #1555. Don't create multiple posts for the same issue.  I have provided some information regarding `draw` parameter in the tagged discussion.

If you'd like further assistance, provide a reproducible GitHub repo or StackBlitz link :)

I'm locking this now as there's no way to close it like in Issues section. :\
